app/controllers/errors_controller.rb

```diff
@@ -72,6 +72,16 @@ def resolve_issues (ids, resolve, errors_per_page, page, current_error)
     @pagination = errors.page(page).per(errors_per_page)
   end
 
+  def current_issue_page(resolved, page, errors_per_page)
+    errors = current_website.grouped_issues.order('last_seen DESC')
+    errors = resolved ? errors.with_status(:resolved) : errors.with_status(:unresolved)
+    if @error.resolved? == resolved
```

Comment:
That's not what I'm saying. I'm saying that you should not be doing that if like that. It should be `if @issue.resolved?` that's all there is no need for `== resolved` what the hell does that do?